Assets/HoloToolkit/Common/Scripts/CrossPlatform/MetroHandleRef.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-#if UNITY_WSA && !UNITY_EDITOR
+#if UNITY_WSA && !UNITY_EDITOR && !ENABLE_IL2CPP
```

Comment:
Not having the same #IF as others?  No checking for NetStandard2?

Reply:
Same as https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2127#discussion_r190957707